### PR TITLE
[ARM CPU] Fix interpolate tests

### PR DESCRIPTION
**Pull request by allnes**
### Details:
 - [ARM32 CI](https://openvino-ci.toolbox.iotg.sclab.intel.com/job/private-ci/job/customJobs/job/build-linux-debian_9_arm/105/)

### CODE REVIEW

File: src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp [hunk 1]
```diff
@@ -81,9 +99,17 @@ bool ov::intel_cpu::ACLInterpolateExecutorBuilder::isSupportedConfiguration(
         const std::vector<MemoryDescPtr> &dstDescs) {
     auto& inp_shape = srcDescs[0]->getShape().getDims();
     auto& out_shape = dstDescs[0]->getShape().getDims();
-
-    float scale_h = static_cast<float>(out_shape[2]) / inp_shape[2];
-    float scale_w = static_cast<float>(out_shape[3]) / inp_shape[3];
+    int index_h, index_w;
+    if (srcDescs[0]->hasLayoutType(LayoutType::ncsp)) {
+        index_h = 2;
+        index_w = 3;
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp:105]: What about other ranks, i.e. rank 2?

Comment (EgorDuplensky) [reply src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp:105]: Discussed, can we add an assert for a rank here?

File: src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp [hunk 2]
```diff
@@ -9,19 +9,25 @@ bool ov::intel_cpu::ACLInterpolateExecutor::init(const InterpolateAttrs &interpo
...
-
-    if ((coord_mode == InterpolateCoordTransMode::pytorch_half_pixel && out_shape[2] > 1 && out_shape[3] > 1) ||
-        coord_mode == InterpolateCoordTransMode::half_pixel) {
+    int index_h, index_w;
+    if (dstDescs[0]->hasLayoutType(LayoutType::ncsp)) {
+        index_h = 2;
+        index_w = 3;
+    } else if (dstDescs[0]->hasLayoutType(LayoutType::nspc)) {
+        index_h = 1;
+        index_w = 2;
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp:22]: Please create a static method to avoid duplication

Comment (allnes) [reply src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp]: @allnes corrected

File: src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp (hunk 1 above)
Comment (allnes) [reply src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp]: ACL interpolate use only rank = 4

File: src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp [hunk 3]
```diff
@@ -33,8 +46,18 @@ bool ov::intel_cpu::ACLInterpolateExecutor::init(const InterpolateAttrs &interpo
             return false;
     }
 
-    auto srcDims = srcDescs[0]->getShape().getStaticDims();
-    auto dstDims = dstDescs[0]->getShape().getStaticDims();
+    auto srcDims = srcDescs[0]->getShape().getDims();
+    auto dstDims = dstDescs[0]->getShape().getDims();
```
Comment (EgorDuplensky) [inline src/plugins/intel_cpu/src/nodes/executors/acl/acl_interpolate.cpp:50]: auto&
Coverity and other analyzers might complain

**Review by EgorDuplensky — APPROVED** [review]

**Pull request merged**